tests/test_turbot.py

```diff
@@ -762,7 +817,18 @@ async def test_on_message_uncollect(self, client, lines):
             "> a foot",
             None,
         )
-        assert lines(turbot.FOSSILS_FILE) == ["author,name\n", "1,ammonite\n"]
```

Comment:
Noticed this bug when I added these tests. Hopefully using more realistic user ids will avoid this kind of bug in the future.